gr-fft/lib/fft.cc

```diff
@@ -104,7 +97,7 @@ static void unlock_wisdom()
 static void import_wisdom()
 {
     auto wisdom_path = gr::paths::cache() / WISDOM_FILENAME;
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || defined(_WIN32)
```

Comment:
Is the conditional code necessary? The `.string().c_str()` version appears to work on Linux at least.